### ISSUE TITLE
Hippocampal malrotation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9557]
**Preferred term label:**
Hippocampal malrotation

**Synonyms**

**Definition (free text, please give PubMed ID)**
See https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8067421/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614402   1/1
PMID:22095910

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Hippocampal malrotation (HIMAL), also termed incomplete inversion of the hippocampus or hippocampal malformation, is an increasingly recognized neuroimaging finding of undetermined clinical significance. It is characterized by features including (i) Round or pyramidal shape instead of ovoid shape; (ii) Medial position of the hippocampus on the hippocampal sulcus; (iii) The collateral sulcus is excessively deep or verticalized; (iv) Fimbria located medial to the hippocampus; (v) Small or displaced fornix; (vi) Enlarged temporal horn and empty choroid fissure; (vii) Thickened subiculum; (viii) Reduced upper horizontal portion of the parahippocampal gyrus.
